src/core/Akka.Remote.Tests/Properties/AssemblyInfo.cs

```diff
@@ -40,4 +42,4 @@
 // [assembly: AssemblyVersion("1.0.*")]
 [assembly: AssemblyVersion("1.0.0.0")]
 [assembly: AssemblyFileVersion("1.0.0.0")]
-
+[assembly: CollectionBehavior(CollectionBehavior.CollectionPerAssembly, DisableTestParallelization = true)]
```

Comment:
When I was running the tests in VS it was running them in parallel and it caused some to fail